R/utils-ui.R

```diff
@@ -1,12 +1,264 @@
-# opening act of an eventual transition away from the ui_*() functions and towards
-# the cli-mediated UI we're using in other packages
+# usethis theme ----------------------------------------------------------------
+usethis_theme <- function() {
+  list(
+    # add a "todo" bullet, which is intended to be seen as an unchecked checkbox
+    ".bullets .bullet-_" = list(
+      "text-exdent" = 2,
+      before = function(x) paste0(cli::col_red(cli::symbol$checkbox_off), " ")
+    ),
+    # historically, usethis has used yellow for this
+    ".bullets .bullet-i" = list(
+      "text-exdent" = 2,
+      before = function(x) paste0(cli::col_yellow(cli::symbol$info), " ")
+    ),
+    # we have enough color going on already, don't add color to `*` bullets
+    ".bullets .bullet-*" = list(
+      "text-exdent" = 2,
+      before = function(x) paste0(cli::symbol$bullet, " ")
+    ),
+    # apply quotes to `.field` if we can't style it with color
+    span.field = list(transform = single_quote_if_no_color)
```

Comment:
I think we really need `.field` to get quoted in the absence of color. That is what `ui_field()` does.